docs/src/pages/index.tsx

```diff
@@ -7,8 +7,6 @@ import { WindowLocation } from "@reach/router";
 
 import { RecentBookmarks } from "../components/Bookmarks";
 // temporarily extract images from @streamlinehq/streamlinehq until they fix the install script
-import LoveBirdIcon from "../images/streamline-light/love-bird.svg";
```

Comment:
Is this image being used somewhere else? Maybe we can remove it too. Same for the other one below